src/main/java/de/tum/in/www1/artemis/service/notifications/SingleUserNotificationService.java

```diff
@@ -81,6 +81,17 @@ public void notifyUserAboutNewAnswerForCoursePost(Post post, Course course) {
         notifyGroupsWithNotificationType(post, NotificationType.NEW_REPLY_FOR_COURSE_POST, course);
     }
 
+    /**
+     * notifyUserAboutPlagiarismCase creates a plagiarismNotification saves it to the database and returns it.
+     * @param plagiarismNotification A singleUserNotification
+     * @return converted plagiarism notification
+     */
+    public SingleUserNotification notifyUserAboutPlagiarismCase(SingleUserNotification plagiarismNotification) {
+        var res = singleUserNotificationRepository.save(plagiarismNotification);
```

Comment:
Sorry, but this seems to break almost every convention of this service. 
If you look at the other methods in this or the respective GroupNotificationService you simply call notifyGroupsWithNotificationType(...) and everything else is done for you by the existing "pipeline" in this service. (I currently also created and merge multiple PRs that change or add new notification types and this is all I ever have to do in such a service^^)